documentation/guides/draft/rust/get-started/xx-cloud-node/README.md

```diff
@@ -0,0 +1,95 @@
+# Using Cloud nodes
+
+## Ockam Hub
+
+In previous examples we learned how to send messages between nodes using [transports](../07-routing-over-transport).
+
+But to connect two nodes at least one of them must be exposed via hostname or a public IP, which makes connecting devices challenging. To address that we can use special kind of node running on the cloud with a fixed hostname and make it forward messages between devices. We call such nodes Ockam Hub Nodes.
+
+You can host a Hub Node yourself, or use one provided by Ockam service at https://hub.ockam.network
+
+Hub Nodes run persistent workers, which are called services. Services can be used for discovery, routing and
+integration with various cloud services used by the application.
```

Comment:
First mention of Services, which we should expand upon. Most workers are persistent, I think a more defining factor is the well-known/apriori knowledge of the workers at known addresses